src/views/Affiliates/community-chart/ProgramChartContainer.tsx

```diff
@@ -49,7 +50,7 @@ export const CommunityChartContainer = () => {
           },
         ]}
       />
-      <$ChartContainer tw="bg-color-layer-3 p-2">
+      <$ChartContainer tw="bg-color-layer-3 p-2 notTablet:rounded-0.5">
```

Comment:
iirc this breakpoint doesn't perfectly match where they collide, right? Does it look okay?

Reply:
Changed to collapse border-radius at 80rem